src/Parser/Rogue/Assassination/CombatLogParser.js

```diff
@@ -0,0 +1,63 @@
+import CoreCombatLogParser from 'Parser/Core/CombatLogParser';
+
+import DamageDone from 'Parser/Core/Modules/DamageDone';
+import Abilities from './Modules/Features/Abilities';
+import AlwaysBeCasting from './Modules/Features/AlwaysBeCasting';
+
+import ComboPointDetails from '../Common/Resources/ComboPointDetails';
+import ComboPointTracker from '../Common/Resources/ComboPointTracker';
+import ComboPoints from './Modules/RogueCore/ComboPoints';
+import EnergyDetails from '../Common/Resources/EnergyDetails';
+import EnergyTracker from '../Common/Resources/EnergyTracker';
+import Energy from './Modules/RogueCore/Energy';
+
+
+
+import EnvenomUptime from './Modules/Spells/EnvenomUptime';
+import GarroteUptime from './Modules/Spells/GarroteUptime';
+import RuptureUptime from './Modules/Spells/RuptureUptime';
+
+
+import MantleOfTheMasterAssassin from '../Common/Legendaries/MantleOfTheMasterAssassin';
+import SoulOfTheShadowblade from '../Common/Legendaries/SoulOfTheShadowblade';
+import InsigniaOfRavenholdt from '../Common/Legendaries/InsigniaOfRavenholdt';
+import DreadlordsDeceit from '../Common/Legendaries/DreadlordsDeceit';
+
+class CombatLogParser extends CoreCombatLogParser {
+  static specModules = {
+    //Feature
+    damageDone: [DamageDone, { showStatistic: true }],
+    abilities: Abilities,
+    alwaysBeCasting: AlwaysBeCasting,
+
+    //Resource
+    comboPointTracker: ComboPointTracker,
+    comboPointDetails: ComboPointDetails,
+    comboPoints: ComboPoints,
+    energyTracker: EnergyTracker,
+    energyDetails: EnergyDetails,
+    energy: Energy,
+
+    //Trackers
+    // mantleDamageTracker: MantleDamageTracker,
```

Comment:
I would remove the dead code since you're planning to leave this for someone else to maintain.

Reply:
Was going to do something with it but left it out of the PR =) removed